swf/models/workflow.py

```diff
@@ -246,7 +246,7 @@ def start_execution(self, workflow_id=None, task_list=None,
         :type   input: dict
 
         :param  tag_list: Tags associated with the workflow execution
-        :type   tag_list: String
+        :type   tag_list: String or list of strings
```

Comment:
Awesome, didn't know! I don't have the time to do that tonight, but I'll definitely add something tomorrow.